engine/session_live.js

```diff
@@ -484,21 +486,14 @@ class SessionLive {
       // Shift the top vod segment on all variants
       for (let i = 0; i < vodBws.length; i++) {
         let seg = this.vodSegments[vodBws[i]].shift();
-        if (seg && seg.discontinuity || seg && seg.cue) {
-          if (seg.discontinuity) {
-            incrementDiscSeqCount = true;
-          }
-          seg = this.vodSegments[vodBws[i]].shift();
-        }
-        if (seg && seg.discontinuity || seg && seg.cue) {
-          if (seg.discontinuity) {
-            incrementDiscSeqCount = true;
-          }
+        if (seg && seg.discontinuity) {
+          incrementDiscSeqCount = true;
           this.vodSegments[vodBws[i]].shift();
         }
       }
       if (incrementDiscSeqCount) {
-        this.mediaSeqCount++;
```

Comment:
This was a variable typo we missed. And `incrementDiscSeqCount` needs to be reset to false so that we don't increment twice no matter what.